src/main/home/send/SendTip/index.tsx

```diff
@@ -206,6 +227,7 @@ const SendTipScreen = ({ route }: SendTipScreenProps) => {
           </TouchableOpacity>
         </View>
         <TouchableOpacity
+          onPress={() => navigateToOverview()}
```

Comment:
no need for arrow function here

```suggestion
          onPress={navigateToOverview}
```